words.json

```diff
@@ -0,0 +1 @@
+{"words": ["secret", "password"]}
```

Comment:
json file should be restructured, Your IDE might have a convenient plugin to do so. json file could be moved to a more suitable folder like `data`